apps/bfd-db-migrator/pom.xml

```diff
@@ -82,15 +149,22 @@
 					</configuration>
 				</plugin>
 				<plugin>
-					<!-- Configure the dependency plugin such that it can be used to grab the
-						 the test dependency artifacts, for manual testing or for use in integration
-						 tests. -->
 					<groupId>org.apache.maven.plugins</groupId>
 					<artifactId>maven-dependency-plugin</artifactId>
 					<version>2.10</version>
+					<!-- Unpack the assembly for use in ITs. -->
 					<configuration>
 						<overWriteReleases>false</overWriteReleases>
 						<overWriteSnapshots>true</overWriteSnapshots>
+						<artifactItems>
+							<artifactItem>
+								<groupId>${project.groupId}</groupId>
+								<artifactId>${project.artifactId}</artifactId>
+								<version>${project.version}</version>
+								<type>zip</type>
+								<outputDirectory>${project.build.directory}/db-migrator</outputDirectory>
+							</artifactItem>
+						</artifactItems>
```

Comment:
This is the configuration that I neglected to add that caused us a little bit of pain, is that right?

Reply:
Indeed!